app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentPresenter.kt

```diff
@@ -216,35 +216,36 @@ class AdyenPaymentPresenter(private val view: AdyenPaymentView,
                 }
                 paymentFailed(it.status) -> {
```

Comment:
I know that this is not from this pr but we should change this method name to match an action.
We should call it **isPaymentFailed**